tests/gaslessTransaction.test.ts

```diff
@@ -0,0 +1,21 @@
+import {VerifiedWallet, Provider} from '../src'
+import {UseropContract} from '../src'
+
+require('dotenv').config();
+
+describe("userop gasless transaction", () => {
+
+    test('gasless transaction should succeed', async() => { 
+        const mnemonics = await VerifiedWallet.generateMnemonic()
+        const wallet = VerifiedWallet.importWallet(mnemonics)
+        const signer = wallet.setProvider(
+          Provider.stackUpProvider(process.env.BUNDLE_URL)
+      )
+        const contract = new UseropContract(wallet.address,signer)
```

Comment:
@herberthk 

I can not understand why you have UserOpContract ? You can just use an existing SDK contract to test.

Reply:
@kallolborah In the telegram chat, I asked you  whether I should create a class contract that extends userop version of `VerifiedContract` that will be instantiated in the tests and you answered "Yes that’s what you've been telling me from day one",
Now I have failed to understand the existing SDK contract to test, I'm I missing something? I'm sorry I have failed to understand. So I'm kindly seeking for more information thanks.

Reply:
@herberthk 

The class contract that extends userop version of VerifiedContract is already there. See below. You just need to instantiate and call the Custody contract function in your test.

```
export default class Custody extends VerifiedContract {

    public contractAddress: string
    
    constructor(signer: VerifiedWallet, contractNetworkAddress: string) {
        
        const address = contractNetworkAddress        
        super(address, JSON.stringify(abi), signer)

        this.contractAddress = address
    }

    public async createVault(_creator: string, _id: string, options?: { gasPrice: number, gasLimit: number }): any {
        await this.validateInput(DATATYPES.STRING, _creator)
        await this.validateInput(DATATYPES.STRING, _id)
        return this.callContract(FUNCTIONS.CREATEVAULT, this.sanitiseInput(DATATYPES.BYTE32, _creator), _id, options)
    }
}
```